doc/publishing-open-data-in-fiware/fiware-extended-ckan.md

```diff
@@ -0,0 +1,117 @@
+# CKAN Monetization Architecture
+
+In the context of the FIWARE platform, CKAN is used as the catalog where
+data can be published, discovered, managed, and consumed. Within the FIWARE
+platform, not only static datasets (CSV, XLSX, etc) are supported, but also
+right time context information can be published in CKAN as Context Broker
+queries (pollutants level in Madrid, etc.)
+
+In addition, FIWARE supports to manage the access and monetize the published
```

Comment:
In addition, using the Biz Ecosystem GE, FIWARE provides the components that allow to manage the access and monetisation of published datasets.